operator/pkg/ciliumendpointbatch/endpointbatch.go

```diff
@@ -275,6 +278,12 @@ func (c *CiliumEndpointBatchController) syncCeb(key string) error {
 			return err
 		}
 	}
+	// Update metrics
+	metrics.CiliumEndpointBatchDensity.WithLabelValues("").Observe(float64(c.Manager.getCepCountInCeb(key)))
+	cepInsert, cepRemove := c.Manager.getCebMetricCountersAndClear(key)
+	metrics.CiliumEndpointsChangeCount.WithLabelValues(metrics.LabelValueCEPInsert).Observe(float64(cepInsert))
+	metrics.CiliumEndpointsChangeCount.WithLabelValues(metrics.LabelValueCEPRemove).Observe(float64(cepRemove))
+	metrics.CiliumEndpointsChangeCount.WithLabelValues(metrics.LabelValueCEPTotal).Observe(float64(cepInsert + cepRemove))
```

Comment:
I'm trying to reason about whether you need the total metric. It adds another set of time series to histograms which are already bloaty. Can you deduce the same data points via a prom query if you removed the total "total" dimension?

Reply:
Sounds good to me, for completeness i had added `total`. We can live without  `total`